backend/.env.test

```diff
@@ -1 +1,3 @@
-JWT_SECRET=GitTestKeyTZGHbb_iuidsuiwew
\ No newline at end of file
+JWT_SECRET=GitTestKeyTZGHbb_iuidsuiwew
```

Comment:
This looks like an actual secret? I hope/assume it's just for testing/development. 
:star: For not committing   sensitive secrets (e.g. API keys) unencrypted to the repository.